packages/vault/src/index.ts

```diff
@@ -81,29 +82,45 @@ export class VaultEnvVars {
       }
     } else {
       // developer's local machine
+      const vaultTokenFile = path.join(os.homedir(), '.vault-token')
       try {
-        const vaultTokenFile = await fs.readFile(path.join(os.homedir(), '.vault-token'), {
-          encoding: 'utf8'
-        })
-        if (vaultTokenFile) {
-          return vaultTokenFile
-        } else if (VAULT_AUTH_GITHUB_TOKEN) {
+        const stats = await fs.stat(vaultTokenFile)
+        const fileExpired = moment().diff(stats.birthtime, 'hours') > 8
```

Comment:
**nitpick**: while the logic makes sense, it might make sense to pull the magic number 8 from a config file or api if we can just in case the token expiry time changes